src/AzurePipelinesToGitHubActionsConverter.Core/Conversion.cs

```diff
@@ -185,30 +181,30 @@ private ConversionResponse ConvertAzurePipelineToGitHubActionV2(string yaml, boo
 
                 //Strategy
                 JsonElement strategy = new JsonElement();
-                if (jsonObject.TryGetProperty("strategy", out jsonElement) == true)
+                if (jsonObject.TryGetProperty("strategy", out jsonElement) )
                 {
                     strategy = jsonElement;
                 }
 
                 //If we have stages, convert them into jobs first:
-                if (jsonObject.TryGetProperty("stages", out jsonElement) == true)
+                if (jsonObject.TryGetProperty("stages", out jsonElement) )
                 {
                     StagesProcessing sp = new StagesProcessing(_verbose);
                     gitHubActions.jobs = sp.ProcessStagesV2(jsonElement, strategy);
                 }
                 //If we don't have stages, but have jobs:
-                else if (jsonObject.TryGetProperty("stages", out jsonElement) == false & jsonObject.TryGetProperty("jobs", out jsonElement) == true)
+                else if (jsonObject.TryGetProperty("stages", out jsonElement) == false && jsonObject.TryGetProperty("jobs", out jsonElement) )
```

Comment:
why not refactor jsonObject.TryGetProperty("stages", out jsonElement) == false to !(jsonObject.TryGetProperty("stages", out jsonElement)) ?

Reply:
that was next! Just did exactly that.
:)